tests/integ/file_system_input_utils.py

```diff
@@ -146,7 +122,9 @@ def _upload_data_and_mount_fs(connected_instance, file_system_efs_id, file_syste
             connected_instance.put(local_file, "temp_tf/")
     connected_instance.put(ONE_P_LOCAL_DATA, "temp_one_p/")
     connected_instance.run(
-        "sudo sh fs_mount_setup.sh {} {} {}".format(file_system_efs_id, file_system_fsx_id, region),
+        "sudo sh fs_mount_setup.sh {} {} {} {} {}".format(
+            file_system_efs_id, file_system_fsx_id, region, EFS_MOUNT_DIRECTORY, FSX_MOUNT_DIRECTORY
```

Comment:
what is the point of passing these in now? it doesn't look like anything has changed to make this necessary.

Reply:
Flora and Lauren talked offline and agreed it is OK to pass the parameters here.